clients/extension/popup.html

```diff
@@ -10,59 +10,55 @@
   <meta charset="utf-8">
   <meta name="author" content="Lighthouse Team">
   <meta name="viewport" content="width=device-width">
-  <title>Lighthouse Report</title>
+  <title>Lighthouse</title>
   <link rel="stylesheet" href="styles/lighthouse.css">
-  <link rel="stylesheet" href="styles/lighthouse-loading.css">
 </head>
 <body>
-
-  <header class="header">
-    <img src="images/lh_logo_bg.png" class="header__icon" width="100" height="100" alt="Lighthouse Logo">
-    <div class="header-titles">
-      <h1 class="header-titles__main">Lighthouse</h1>
-      <h2 class="header-titles__url">...</h2>
-      <div class="header-titles__throttling">
-        <label for="lantern-checkbox">
-          <input
-            id="lantern-checkbox"
-            type="checkbox"
-            checked />
-          Simulated throttling (faster)
-        </label>
-      </div>
-    </div>
-  </header>
-
   <main class="main" role="main">
-    <div class="feedback"></div>
-
-    <button class="button button--configure" id="configure-options">Options</button>
-    <button class="button button--generate" id="generate-report">Generate report</button>
-  </main>
-
-  <!-- Show running view initially (.subpage--visible). -->
-  <aside class="status subpage subpage--visible">
-    <div class="lighthouse-effects">
-      <img src="images/lh_logo_bg_no-light.png" alt="LH logo">
-      <div class="lighthouse-effects__wrapper">
-        <div class="lighthouse-effects__light"></div>
+    <div class="section section--run">
+      <img src="images/lh_logo_128x128.png" class="header__icon" width="75" height="75" alt="Lighthouse Logo">
+      <button class="button button--generate">Generate report</button>
+      <div role="button" class="button button--configure">
+        <svg width="25" height="25" viewBox="0 0 14 15">
+          <path fill="#fff" d="M-1-1h16v17H-1z" />
+          <path d="M12.526 8.07c.029-.224.05-.448.05-.686 0-.238-.021-.462-.05-.686l1.518-1.155a.347.347 0 0 0 .087-.448l-1.44-2.422a.367.367 0 0 0-.439-.154l-1.791.7a5.295 5.295 0 0 0-1.217-.686L8.971.678a.348.348 0 0 0-.353-.294H5.74a.348.348 0 0 0-.353.294l-.273 1.855a5.568 5.568 0 0 0-1.216.686l-1.792-.7a.356.356 0 0 0-.44.154L.228 5.095a.339.339 0 0 0 .087.448l1.518 1.155c-.029.224-.05.455-.05.686 0 .23.021.462.05.686L.314 9.225a.347.347 0 0 0-.087.448l1.44 2.422c.086.154.28.21.439.154l1.792-.7c.374.28.777.51 1.216.686l.273 1.855a.348.348 0 0 0 .353.294h2.878c.18 0 .331-.126.353-.294l.273-1.855a5.568 5.568 0 0 0 1.217-.686l1.792.7c.165.063.352 0 .438-.154l1.44-2.422a.347.347 0 0 0-.087-.448L12.526 8.07zM7.179 9.834c-1.389 0-2.519-1.1-2.519-2.45 0-1.351 1.13-2.45 2.52-2.45 1.388 0 2.518 1.099 2.518 2.45 0 1.35-1.13 2.45-2.519 2.45z" />
+        </svg>
       </div>
+      <span class="psi-disclaimer"><a href="https://developers.google.com/speed/docs/insights/v5/get-started?utm_source=lh-chrome-ext" target="_blank" rel="noopener nofollow">Uses the PSI API</a></span>
```

Comment:
can you also set the URL (maybe minus the utm_source) as the title?

since there's no status bar, it's nice to preview the link somehow before clicking it.